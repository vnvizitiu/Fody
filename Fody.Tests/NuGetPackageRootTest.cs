using System.IO;
using System.Linq;
using NUnit.Framework;
using ObjectApproval;

[TestFixture]
public class NuGetPackageRootTest
{
    [Test]
    public void WithNuGetPackageRoot()
    {
        var combine = Path.GetFullPath(Path.Combine(TestContext.CurrentContext.TestDirectory, "../../FakeNuGetPackageRoot"));
        var nuGetPackageRoot = Path.GetFullPath(combine);
        var result = AddinFinder.ScanNuGetPackageRoot(nuGetPackageRoot)
            .Select(s=>s.Replace(@"\\", @"\").Replace(combine, "")).ToList();
        ObjectApprover.VerifyWithJson(result);
    }

}