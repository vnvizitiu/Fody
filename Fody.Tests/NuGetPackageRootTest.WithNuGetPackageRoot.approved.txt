[
  "\\Weaver1.Fody\\1.0.7\\Weaver1.Fody.dll",
  "\\Weaver1.Fody\\1.0.8\\Weaver1.Fody.dll",
  "\\weaver2.fody\\2.0.7\\Weaver2.Fody.dll",
  "\\weaver2.fody\\2.0.8\\Weaver2.Fody.dll"
]